evoxbench/test_suites/citymop.py

```diff
@@ -0,0 +1,38 @@
+from evoxbench.benchmarks import MoSegNASBenchmark
+
+__all__ = ['citymop']
+
+
+def citymop(problem_id):
+    if problem_id == 1:
+        return MoSegNASBenchmark(
+            objs = 'params&flops&mIoU&latency&FPS',
+            normalized_objectives=False
+        )
+    elif problem_id == 2:
+        return MoSegNASBenchmark(
+            objs = 'params&mIoU&latency&FPS',
+            normalized_objectives=False
+        )
+    elif problem_id == 3:
+        return MoSegNASBenchmark(
+            objs = 'flops&mIoU&latency&FPS',
+            normalized_objectives=False
+        )
+    elif problem_id == 4:
+        return MoSegNASBenchmark(
+            objs = 'params&flops&mIoU',
+            normalized_objectives=False
+        )
+    elif problem_id == 5:
+        return MoSegNASBenchmark(
+            objs = 'mIoU&latency&FPS',
+            normalized_objectives=False
+        )
+    elif problem_id == 6:
```

Comment:
Is problem 6 a single objective problem? In **EvoXBench**, the number of objective need to be greater than or equal to 2. The order of the six test problems can be adjusted according to the objective number from least to most.

Reply:
Thanks for the correction! The citymop problem objective file has been modified according to the advice.

Reply:
Do I have to merge the commits together again?